### ISSUE TITLE
Implement Match

### ISSUE THREAD

**Issue by kevinburke** [kevinburke/ssh_config#6]

**Comment by kevinburke** [comment]
Punting on this for the moment

**Comment by Mech45** [comment]
Hey, so today it raised an error when he finds a "Match" directive pattern, maybe can we just pass through it without raise anything ?
Thanks for your package by the way :)

**Comment by cv** [comment]
@kevinburke would you consider a PR that made it an option to skip `Match exec` blocks entirely? I don't need to parse those at the moment, but the rest of the ssh config is useful to me.

**Comment by gromero** [comment]
@kevinburke ping (about @cv question). I think it's still valid and need to handled. Thanks.

**Comment by quite** [comment]
> @kevinburke ping (about @cv question). I think it's still valid and needs to be handled. Thanks.

Yes I agree. While #45 (#38) does help a little, we loose a lot of functionality that is provided by the `UserSetting`s type. So I think a way to skip Match lines at a higher level would be good, as long as Match is unsupported and results in failure to parse config -- a bool in `UserSettings`?

**Comment by iamFrancescoFerro** [comment]
+1 on @quite suggestion

**Comment by iamFrancescoFerro** [comment]
@quite @kevinburke 

Would that be ok ?

https://github.com/kevinburke/ssh_config/pull/65

**Comment by quite** [comment]
@kevinburke nice, looking forward a release, and updated CHANGELOG so it's clear what is and is not supported in the release. Great work!
